staubli_rx160_support/urdf/rx160l_macro.xacro

```diff
@@ -68,7 +68,7 @@
 			<visual>
 				<origin xyz="0 0 0" rpy="0 0 0"/>
 				<geometry>
-				    <mesh filename="package://staubli_rx160_support/meshes/rx160l/visual/link_4.stl"/>
+					<mesh filename="package://staubli_rx160_support/meshes/rx160l/visual/link_4.stl"/>
```

Comment:
I think we're confusing each other here.

 1. currently, all xacros (and launch files, etc) in the `staubli` repository use tabs for indentation
 1. a commit in #10 introduced a file that mixed spaces and tabs
 1. commit c0052c3164b8fe30b1831c971f836b20e9d14545 fixes that: it replaces the (3) spaces with a single tab character
 1. once merged, all files will only be using tabs again
 1. then #9 can be fixed, and only spaces will be used

I hope this clears things up.